addons/product/static/src/scss/report_label_sheet.scss

```
@@ -68,7 +68,7 @@
     }
 
     // generic 4x12 label w/ all same size font
-    div.o_label_4x12 {
+    div.o_label_4x12, div.o_label_4x7 {
```

Regarding my previous comment:
```suggestion
    .o_label_4x12 {
```
- Remove `o_label_4x7` since we want to keep the 4x12 format (and `o_label_4x7` wasn't used anymore anyway ?)
- Remove the `div` so the `o_label_4x12` rules aren't limited to div (I don't think it was wanted ? cc @ticodoo ) and the product's name and lot will be written smaller